components/brave_new_tab_ui/actions/new_tab_actions.ts

```diff
@@ -71,4 +71,6 @@ export const showSettingsMenu = () => action(types.NEW_TAB_SHOW_SETTINGS_MENU)
 
 export const closeSettingsMenu = () => action(types.NEW_TAB_CLOSE_SETTINGS_MENU)
 
-export const toggleShowBackgroundImage = () => action(types.NEW_TAB_TOGGLE_SHOW_BACKGROUND_IMAGE)
+export const preferencesUpdated = (prefs: any) => action(types.NEW_TAB_PREFERENCES_UPDATED, {
```

Comment:
should this be `any`? I know CR has some weird types but they aren't usual

Reply:
oh good spot, there is a type for this now, `type Preferences`

Reply:
This is fixed - great spot